src/plugin.ts

```diff
@@ -100,6 +102,14 @@ export const trackChangesPlugin = (opts: TrackChangesOptions = { userID: 'anonym
       trs.forEach((tr) => {
         const wasAppended = tr.getMeta('appendedTransaction') as Transaction | undefined
         const skipMetaUsed = skipTrsWithMetas.some((m) => tr.getMeta(m) || wasAppended?.getMeta(m))
+
+        // track changes allows free reign for client sync, because, if the changes was supposed to be tracked it should've been done on the respective client.
+        const collabRebased = tr.getMeta('rebased')
```

Comment:
I saw it being used in some places and with that I don't see anywhere in the plugin code that it would stop tracking for such transaction so I assumed it is mean to be used for something else such as actions history (ctrl+z operated)

Reply:
so this is will be true just if the rebased is 0 which is the size of [unconfirmed](https://github.com/ProseMirror/prosemirror-collab/blob/c019e4cd1e05504d403d98e6bfec67fe1a80c895/src/collab.ts#L128) slice in the step

Reply:
So presence of 'rebased' in meta means that this transaction brings in the steps that came from the server and that transaction shouldn't be processed but only track changes has to be recalculated

Reply:
thanks for explanation got it